src/main/java/com/kainos/ea/backend/repositories/BandCompetencyRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.kainos.ea.backend.repositories;
+
+import com.kainos.ea.backend.models.BandCompetency;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface BandCompetencyRepository extends CrudRepository<BandCompetency, String> {
+
+    public List<BandCompetency> findAllByBand_BandName(String bandName);
```

Comment:
Is the underscore necessary?

Reply:
https://stackoverflow.com/a/55579268
I got it from here